ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs

```diff
@@ -38,6 +41,40 @@ public ReactTextShadowNode()
         }
 
         /// <summary>
+        /// Sets the TextDecorationLine for the node.
+        /// </summary>
+        /// <param name="textDecorationLineValue">The TextDecorationLine value.</param>
```

Comment:
Agree with @matthargett. I've just tried to have autodocs style seem like other similar properties, like FontStyle and LineHeight